patches/server/0035-Disable-thunder-and-add-chance-config.patch

```diff
@@ -1,33 +1,36 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Sudzzy <originmc@outlook.com>
 Date: Wed, 2 Mar 2016 14:52:43 -0600
-Subject: [PATCH] Disable thunder
+Subject: [PATCH] Disable thunder and add chance config
 
 
 diff --git a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
-index cac8606f65e587d34e2ae12f5962f236c316751e..b18622c199f57e0d7c201582fe83bb75dca143f9 100644
+index cac8606f65e587d34e2ae12f5962f236c316751e..82c0bd82e91b3124732d927f554a6675843dca95 100644
 --- a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
 +++ b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
-@@ -171,4 +171,9 @@ public class PaperWorldConfig {
+@@ -171,4 +171,12 @@ public class PaperWorldConfig {
      private void disableExplosionKnockback(){
          disableExplosionKnockback = getBoolean("disable-explosion-knockback", false);
      }
 +
++    public int thunderChance;
 +    public boolean disableThunder;
```

Comment:
the different order here compared to the method below bothers me more than I would like to admit :D